.devcontainer/images/Dockerfile

```
@@ -0,0 +1,291 @@
+    rm -rf /var/lib/apt/lists/*
+
+# CACHE_BUST_DYNAMIC: Forces cache invalidation for all dynamic installations.
+# Set to current date (YYYY-MM-DD) during scheduled builds to pull latest versions.
+# See: https://docs.docker.com/build/cache/invalidation/
+ARG CACHE_BUST_DYNAMIC=stable
+ARG GITHUB_TOKEN=
+
+# =============================================================================
+# PPA-Based Tools (HashiCorp, 1Password, Ansible) - consolidated
+# =============================================================================
+RUN curl -fsSL https://apt.releases.hashicorp.com/gpg | gpg --dearmor -o /usr/share/keyrings/hashicorp-archive-keyring.gpg && \
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)